samples/app-dev/Flow-Design-Concepts/allOf_oneOf-anyOf_construct_schema/readme.md

```diff
@@ -26,20 +28,27 @@ The anyOf keyword allows for validation against multiple schemas. If the data va
+![allOf configuration](../../import-screenshots/allOfProperties.png)
+![Enter the POST request schema](../../import-screenshots/allOf_request_payload.png)
+![verify the POST response schema](../../import-screenshots/allOf_response_payload.png)
+
+10. Click on *Invoke_OneOfKeywordAPISpecConfiguredFlow* click on mapper activity,navigate to input tab and expand all configurations.Click on oneOf schema and select the radio button corrosponding to required schema.Save the configuration and verify that in mapper input tab,properties of selected schema are appeared in activity inputs.Verify request and response payload for oneOf schema.
+![oneOf configuration](../../import-screenshots/selectOneOfSchema.png)
 ![select oneOf configuration](../../import-screenshots/select_oneOf_configuration.png)
-![verify mapper activity for oneOf schema properties](../../import-screenshots/oneOf_mapper_schema_properties.png)
+![Enter the oneOf POST request schema](../../import-screenshots/oneOf_request_payload.png)
+![Verify the oneOf POST response schema](../../import-screenshots/oneOf_response_payload.png)
 
-9. Click on *anyOfsample* click on mapper activity,navigate to input tab and expand all configurations.Click on anyOf schema and select the checkboxes(multiple schemas can be selected) corrosponding to required schema.Save the configuration and verify that in mapper input tab,properties of selected schema are appeared in activity inputs.
-![anyOf configuration](../../import-screenshots/anyOf_configuration.png)
+11. Click on *Invoke_AnyOfKeywordAPISpecConfiguredFlow* click on mapper activity,navigate to input tab and expand all configurations.Click on anyOf schema and select the checkboxes(multiple schemas can be selected) corrosponding to required schema.Save the configuration and verify that in mapper input tab,properties of selected schema are appeared in activity inputs.
+![anyOf configuration](../../import-screenshots/selectAnyOfSchema.png)
 ![select anyOf configuration](../../import-screenshots/select_anyOf_configuration.png)
-![verify mapper activity for anyOf schema properties](../../import-screenshots/oneOf_mapper_schema_properties.png)
+![Enter the anyOf POST request schema](../../import-screenshots/anyOf_request_payload.png)
+![Verify the anyOf POST response schema](../../import-screenshots/anyOf_response_payload.png)
 
-10. Fill the mapper inputs and verify response in flow tester
 
-**Note:- For oneOf schema we have used schema having oneOf keyword, similarly for anyOf .**
-**Note:- Schema constructs can be used where schema is being used eg- Mapper and Invoke REST Service activity and Receive HTTP Message trigger.**
 **Note:- Return and Reply to Trigger activity does not possess selecting oneOf-anyOf schema since it is driven by trigger reply of Receive HTTP Message .**
```

Comment:
Add Output section and add screenshot for each of service with request payload and output